packages/UI/.storybook/preview.jsx

```diff
@@ -66,7 +66,7 @@ export const decorators = [
   ),
 ];
 export const parameters = {
-  actions: { argTypesRegex: "^on[A-Z].*" },
+  actions: { argTypesRegex: '^on.*' },
```

Comment:
This change will make storybook observe all the events that start with `on` and display/console log them on the actions tab when the story viewer fires that event by interacting with the component.

pretty helpful to know what events a component supports with this
Example: someone wants to know what is the event responsible for changing the component state from checked to unchecked, they go and interact with the component to trigger that behavior then get to know the event that was fired from the actions tab.